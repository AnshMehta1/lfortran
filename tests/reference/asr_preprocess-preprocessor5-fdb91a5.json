{
    "basename": "asr_preprocess-preprocessor5-fdb91a5",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor5.f90",
    "infile_hash": "96309887c38d452524fc1048a69f630c58fd72edb53964e858ffd7dd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor5-fdb91a5.stdout",
    "stdout_hash": "83575285294c5503339f66007c9d9fff366ad48bc69dc5b4632ba204",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}