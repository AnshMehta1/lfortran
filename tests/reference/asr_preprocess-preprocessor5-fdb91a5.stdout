(TranslationUnit
    (SymbolTable
        1
        {
            preprocessor5:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    preprocessor5
                    []
                    [(Print
                        [(IntegerConstant 1 (Integer 4) Decimal)]
                        ()
                        ()
                    )
                    (Print
                        [(IntegerConstant 12 (Integer 4) Decimal)]
                        ()
                        ()
                    )
                    (Print
                        [(IntegerConstant 3 (Integer 4) Decimal)]
                        ()
                        ()
                    )
                    (Print
                        [(IntegerConstant 6 (Integer 4) Decimal)]
                        ()
                        ()
                    )
                    (Print
                        [(IntegerConstant 15 (Integer 4) Decimal)]
                        ()
                        ()
                    )]
                )
        })
    []
)
